src/style.css

```diff
@@ -2,0 +1,29 @@
+body{
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-image: url('./images/star.svg');
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
```

Comment:
Be consistent with your margins. It's better to add only top or bottom margins, but not both, to avoid potential margin collapse. Here, you are adding a left margin which is fine as long as you are consistent with it throughout your CSS.